fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/DruidClientConfigHelper.java

```diff
@@ -155,6 +236,10 @@ public static DruidServiceConfig getNonUiServiceConfig() {
         return new DruidServiceConfig("Broker", getDruidNonUiUrl(), getDruidNonUiTimeout(), getDruidNonUiPriority());
```

Comment:
Shouldn't `getNonUiServiceConfig()` and `getUiServiceConfig()` be deprecated as well?

Also `getNonUiServiceConfig()` is used in `AsyncDruidWebServiceImplSpec`, which should probably be replaced with calls to `getServiceConfig()`.